### PR TITLE
BCK-3639 Adding description and owner name fields to jobs

### PR DESCRIPTION
**Pull request by aimeemorgan**
This commit adds backend support for description and job owner name fields, plus support for viewing and editing these fields via the Chronos web UI. Changes have been tested via an AWS Chronos instance.

### CODE REVIEW

**Comment by aimeemorgan** [comment]
@mngo87 Please review.

File: src/main/resources/assets/app/scripts/templates/job_detail_view.hbs [hunk 1]
```diff
@@ -19,6 +19,21 @@
       </div>
       <span class="help-inline hide"></span>
     </div>
+    
+     <div class="row-fluid control-group job-detail-row job-detail"
+         data-toggle="tooltip"
+         data-html="true"
+         data-title="Description of the job."
+         data-placement="bottom">
+      <div class="detail">Description</div>
```
Comment (mngo87) [inline src/main/resources/assets/app/scripts/templates/job_detail_view.hbs:28]: NIT: indent one more space.  the rest of the file indents in 2 spaces, so let's keep it consistent.  The same applies to the rest of the div block.

File: src/main/resources/assets/app/scripts/templates/job_detail_view.hbs [hunk 2]
```diff
@@ -30,7 +45,7 @@
            data-rv-text="job.command">{{command}}</div>
       <div class="field cmd">
         <input class="job-input" value="{{command}}"
-               data-rv-value="job.command" name="command" tabindex="2">
+               data-rv-value="job.command" name="command" tabindex="3">
```
Comment (mngo87) [inline src/main/resources/assets/app/scripts/templates/job_detail_view.hbs:48]: Good job on picking up on this subtle detail.

File: src/main/resources/assets/app/scripts/templates/job_detail_view.hbs [hunk 3]
```diff
@@ -55,20 +70,40 @@
...
+          <div class="control-group job-detail"
+               data-toggle="tooltip"
+               data-title="Owner's email address(es)<br>The owner(s) will be emailed in the event of a job failure or disruption."
+               data-html="true"
+               data-placement="bottom">
+            <div class="detail">Owner(s)</div>
+            <div class="display hightlight owner"
+                 data-rv-text="job.owner">{{owner}}</div>
+            <div class="field cmd">
+              <input class="job-input" value="{{owner}}" name="owner" tabindex="6"
```
Comment (mngo87) [inline src/main/resources/assets/app/scripts/templates/job_detail_view.hbs]: looks like the tab index should be 5, please check the rest to see if they line up as well.

**Comment by mngo87** [comment]
Looks good!  Please address the comments and FFTM.

**Comment by aimeemorgan** [comment]
Thanks for the review @mngo87. I addressed your comments and am going to squash commits before merging.

**Pull request closed without merging**